requirements.txt

```diff
@@ -4,3 +4,4 @@ tensorflow==2.5.0
 torch==1.9.0
 tqdm==4.62.0
 transformers==4.9.1
+sacrebleu
```

Comment:
Can you please specify a fixed version for this package?

Reply:
specified :)